splunk-otel-android/build.gradle

```diff
@@ -11,8 +11,6 @@ android {
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 30
-        versionCode 1
-        versionName project.version
```

Comment:
Were those settings not needed?

Reply:
Apparently not. Android studio had a suggestion to remove them since they aren't used by the latest Android gradle plugin.